src/components/Hero/Hero.js

```diff
@@ -4,7 +4,7 @@ import "./Hero.css";
 
 const Hero = ({info}) => {
     return (
-        <div className='Hero' style={{ backgroundImage:`url(${info.backdrop_path})`}}>
+        <div className='hero' style={{ backgroundImage:`url(${info.backdrop_path})`}}>
```

Comment:
Those darn classes and classes! Must have run out of names when they came up with that!